app/styles/_virtual-machine-row.less

```diff
@@ -1,3 +1,7 @@
 .bar-separated + .bar-separated::before {
   content: ' | '
 }
+
+.grid-align {
+  display: inline-grid;
```

Comment:
This isn't supported in IE11/Edge without a vendor prefix.  It's probably best to find an alternate solution.

Reply:
rendered inline, so no more needed